src/Core/Dirt/Entities/OrganizationReport.cs

```diff
@@ -9,11 +9,13 @@ public class OrganizationReport : ITableObject<Guid>
 {
     public Guid Id { get; set; }
     public Guid OrganizationId { get; set; }
-    public DateTime Date { get; set; }
     public string ReportData { get; set; } = string.Empty;
     public DateTime CreationDate { get; set; } = DateTime.UtcNow;
 
     public string ContentEncryptionKey { get; set; } = string.Empty;
+    public string SummaryData { get; set; } = string.Empty;
+    public string ApplicationData { get; set; } = string.Empty;
```

Comment:
❓ I didn't notice this before but why are we initializing these with an empty string? I'd prefer null in these columns until there's real data to store.

Reply:
Yeah we can do that. I'll update the SQL constraint to be 'NULL' instead of 'NOT NULL'